public/assets/fortmatic.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_2" data-name="Layer 2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 33 33"><defs><style>.cls-1{fill:none;stroke:#fff;stroke-miterlimit:10;}</style></defs><g id="Fortmatic"><path class="cls-1" d="M12.87,12.83h11V9.24H9.15V23.76h3.72Z"/><path class="cls-1" d="M16.5,16.5h7.35v3.76a3.51,3.51,0,0,1-3.51,3.5h-.17V20.13H16.5Z"/></g></svg>
```

Comment:
let's either pull these into the component if we think every consumer wants to use them, or else pull the usage of them into this repo, and eg. add them using css.